### ISSUE TITLE
macOS Catalina permission prompt

### ISSUE THREAD

**Issue by michelf** [michelf/sim-daltonism#13]
Perhaps we should add a pre-prompt alert [like xScope is doing](https://blog.iconfactory.com/2019/08/xscope-catalina-and-screen-recording/). One that explains why we need the "screen recording" capability for the pass-through filter even though we aren't actually recording anything.

**Comment by michelf** [comment]
A minimalist idea would be to add a note in the preferences, as this is a likely place for someone to look for when something is amis.

**Comment by mrmetz** [comment]
Yes please. I thought the app was broken, almost uninstalled it. @michelf 's suggestion is already a good one from UX perspective. It is exactly where I was looking, cause I was confused that I was looking at my laptop background.
